src/main/kotlin/fi/metatavu/timebank/api/persistence/model/TimeEntry.kt

```diff
@@ -38,6 +38,9 @@ class TimeEntry {
     @Column
     var updatedAt: OffsetDateTime? = null
 
+    @Column
```

Comment:
Should this be marked with @ManyToOne annotation since there can be multiple TimeEntry instances that refer to one calendar?